vitrina/orgs/urls.py

```diff
@@ -1,10 +1,12 @@
 from django.urls import path
 
 from vitrina.orgs.views import OrganizationListView
+from vitrina.orgs.views import OrganizationSearchResultsView
 
 urlpatterns = [
     # @RequestMapping("/organizations")
     path('organizations/', OrganizationListView.as_view(), name='organization-list'),
+    path('organizations/search/', OrganizationSearchResultsView.as_view(), name='organization-search-results')
```

Comment:
There is no need to add separate search view, search should be enabled if GET `q` parameter is given. See: https://data.gov.lt/organizations?q=ministerija